DataFormats/BTauReco/interface/PixelClusterTagInfo.h

```diff
@@ -0,0 +1,101 @@
+#ifndef DataFormats_BTauReco_PixelClusterTagInfo_h
+#define DataFormats_BTauReco_PixelClusterTagInfo_h
+
+#include "DataFormats/Common/interface/CMS_CLASS_VERSION.h"
+#include "DataFormats/BTauReco/interface/BaseTagInfo.h"
+
+#include "DataFormats/PatCandidates/interface/Jet.h"
+#include "DataFormats/VertexReco/interface/Vertex.h"
+
+namespace reco {
+
+struct PixelClusterProperties {
+    float x = 0;
+    float y = 0;
+    float z = 0;
+    int charge = 0;
+    int layer = 0;
+};
+
+struct PixelClusterData {
+    char L1_R004 = 0;
```

Comment:
Curiosita`: perche` non array 2D?

Reply:
Mi sembra che singoli valori siano piu' gestibili di array. Inoltre la NN che e' stata sviluppata utilizza informazioni scritte piu' o meno allo stesso modo. Magari in futuro, dopo studi ben piu' approfonditi di quelli attuali, si puo' decidere di ampliare o cambiare il modo in cui scriviamo le informazioni.